source/dofs/dof_renumbering.inst.in

```diff
@@ -246,13 +246,19 @@ for (deal_II_dimension : DIMENSIONS; deal_II_space_dimension :  SPACE_DIMENSIONS
      const Point<deal_II_dimension>&,
      const bool);
 
+    template
+    void downstream
+    (DoFHandler<deal_II_dimension>&,
```

Comment:
I think you need instantiations for codim>0 and hp::DoFHandler (assuming that those work with the old inline function).

Reply:
No, they never worked. `downstream` calls `compute_downstream`, which sits in the `.cc` file and was never instantiated for co-dimension DoFHandlers.

But I'm willing to do this in a follow-up, if you want.

Reply:
> No, they never worked.

I am not surprised. ;-)

> But I'm willing to do this in a follow-up, if you want.

Not urgent, there is a pile of other things worth fixing before that.